### ISSUE TITLE
Ugly thing in the BIB rendering

### ISSUE THREAD

**Issue by martindholmes** [TEIC/TEI#1598]
The first item in the Bibliography is this:

~~~~~
No source. made up for the Guidelines. ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵ ↵
~~~~~

where each arrow is a link to the item. The caption is badly punctuated, and cursory at best; that should definitely be fixed. Whether we actually need a list of links to all the examples that happen to have been made up for the Guidelines, especially when the links themselves are completely uninformative, is also worth thinking about.

**Comment by lb42** [comment]
The list of links is a result of the desire to indicate somehow that (we know that) there is no source for an example, which is not of course the same as not specifying a source (because we don't know what it is, or because we forgot to). By all means improve the text and propose a better way of indicating the status of the sources concerned, but don't get rid of the information.

**Comment by martindholmes** [comment]
When the link is just an arrow, though, and there's no identifying information, it seems a bit half-hearted to me. What is the use case for someone reading the BIB page and clicking on one of these links? I want to see a random example of anything at all, but it must be an example that was made up purely for the Guidelines?

**Comment by lb42** [comment]
The back links are there because if you are reading the text of the GL and look up the bib ref for an example you need a way of getting back from the Biblio to wherever you were. They are less important now that we have cute little popups, but they still serve a useful function. You could certainly suppress all but the first three or four of them I suppose, but that might cause confusion if you happen to reach an entry from the sixth or seventh time it's cited. This is unlikely to happen except for the "No source" entry of course, so maybe it doesn't matter.

**Comment by martindholmes** [comment]
We do allow for graceful degradation in the popup system, so people without JS turned on will still follow the link as in the old days. But in this particular case (unsourced examples), they'll follow it to a long undifferentiated list of links with no way to determine which was the one they came from. And perhaps more importantly, for this use case, _they have a Back button_.
I think if we elect to retain all these links, we need to provide some useful information in a title attribute that enables you to either a) find the one you came from, in case your Back button isn't working, and/or b) find an interesting example you want to see, assuming you're searching for something specific.

**Comment by sydb** [comment]
Council decided to move the NONE entry to the bottom of the list and use a comma instead of a period.

**Comment by sydb** [comment]
Partial fix agreed upon on conference call checked-in at 379c66fed3d33c531116cd82b03854176f6c686e.

**Comment by sydb** [comment]
Partial fix agreed upon on conference call checked-in at 379c66fed3d33c531116cd82b03854176f6c686e. Mr. Jenkins [complained on build 2558](http://teijenkins.hcmc.uvic.ca/job/TEIP5-Documentation-dev/2558/parsed_console/), but it seems to have failed on PDF generation, and I do not see why at all. Any thoughts?

**Comment by martindholmes** [comment]
It failed on both Jenkins servers, so it's not an anomaly. I updated my teic/teidev-docker container and ran the build on there, and it failed with exactly the same error:

~~~~~
ANT_OPTS="-Xss2m -Xmx752m -Djava.awt.headless=true" ant -q -lib Utilities/lib/saxon9he.jar -f antbuilder.xml -DXSL=/tei/TEI/P5/../../Stylesheets -DXELATEX=xelatex pdfonce
     [echo] make LaTeX and PDF

BUILD FAILED
/tei/TEI/P5/antbuilder.xml:158: exec returned: 1

Total time: 53 seconds
make: *** [Guidelines.pdf] Error 1
~~~~~

No idea what the problem is at this point, though.

**Comment by sydb** [comment]
I cannot get that far building locally, as I don’t have XeTeX. Using `teic/teidev-docker` container I got the same error, too. I saw that line 158 of `/tei/TEI/P5/antbuilder.xml` is
~~~~~xml
    <exec executable="${XELATEX}" failonerror="true">
      <arg value="-interaction=nonstopmode"/>
      <arg value="Guidelines.tex"/>
    </exec>
~~~~~
So I tried running `xelatex -interaction=nonstopmode Guidelines.tex` on the docker commandline. The console output of that command is 233 KiB of various minor things (underfull boxes) and some that look significant (“Failed to convert input string to UTF16”), but the command worked, I ended up with a readable Guidelines.pdf file. (Although the font for headings looks awful, overall it looks pretty good. And links work.)

So I don’t know what is going on. **But** I did discover that the “none” entry I had moved was inadvertently moved to the end of a different `<listBibl>`. I fixed that screw-up and the error went away in my docker. I don’t get it, but … pushed as 2b15838.
